src/services/Listener/Listener.service.jsx

```diff
@@ -27,6 +27,7 @@ const Listener = props => {
 
           //Item was purchased over 24 hours ago
           if (timeDifference >= 86400) {
```

Comment:
Same suggestion about creating a self documenting constant so that you can eliminate the comment and magic number.
```suggestion
const secondsInDay = 86400;
          if (timeDifference >= secondsInDay) {
```
https://github.com/the-collab-lab/tcl-11-smart-shopping-list/pull/34/files#r486607394
https://github.com/the-collab-lab/tcl-11-smart-shopping-list/pull/34/files#r486624200